dotCMS/src/integration-test/java/com/dotmarketing/servlets/ShortyServletAndTitleImageTest.java

```diff
@@ -0,0 +1,205 @@
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.contentlet.model.ContentletVersionInfo;
+import com.dotmarketing.portlets.contentlet.model.IndexPolicy;
+import com.dotmarketing.portlets.fileassets.business.FileAssetAPI;
+import com.dotmarketing.portlets.folders.model.Folder;
+import com.liferay.portal.model.User;
+
+public class ShortyServletAndTitleImageTest {
+    
+
+    final static byte[] pngPixel=Base64.getDecoder().decode("iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNk+M9QDwADhgGAWjR9awAAAABJRU5ErkJggg==");
+    final static byte[] gifPixel=Base64.getDecoder().decode("R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==");
+    final static byte[] txtBlah = "Here is a text blah".getBytes();
+    private static User user ;
+    private static Host host ;
+    
+    @BeforeClass
+    public static void prepare()  {
+        try {
+            IntegrationTestInitService.getInstance().init();
+            user = APILocator.systemUser();
+            host = APILocator.getHostAPI().findDefaultHost(APILocator.systemUser(), false);
+        } catch (Exception e) {
+            throw new DotRuntimeException(e);
+        }
+
+
+    }
+
+    
+    @Test
+    public void test_getTitleImage() throws Exception{
+        final File txtFile=File.createTempFile("tmp", ".txt");
+        Files.write(txtFile.toPath(), txtBlah);
+        final File pngFile=File.createTempFile("tmp", ".png");
+        Files.write(pngFile.toPath(), pngPixel);
+        final File gifFile=File.createTempFile("tmp", ".gif");
+        Files.write(gifFile.toPath(), gifPixel);
+        final Folder folder = new FolderDataGen().nextPersisted();
+        Contentlet fileAsset = new FileAssetDataGen(folder, pngFile).nextPersisted();
+        
+        assertTrue("we HAVE a title image  ", fileAsset.getTitleImage().isPresent());
+        
+        byte[] fileContents = Files.readAllBytes(fileAsset.getBinary(fileAsset.getTitleImage().get().variable()).toPath());
+
+        assertTrue("TitleImage read correctly", Arrays.equals(fileContents ,pngPixel));
+        
+        fileAsset = APILocator.getContentletAPI().checkout(fileAsset.getInode(), user, false);
+        
+        fileAsset.setIndexPolicy(IndexPolicy.FORCE);
+        fileAsset.setStringProperty(FileAssetAPI.FILE_NAME_FIELD, txtFile.getName());
+        fileAsset.setBinary(FileAssetAPI.BINARY_FIELD, txtFile);
+        
+        fileAsset = APILocator.getContentletAPI().checkin(fileAsset, user, false);
+        
+
+        assertTrue("we DO NOT HAVE a title image as a text file cannot have an image", !fileAsset.getTitleImage().isPresent());
+        
+        
+        fileContents = Files.readAllBytes(fileAsset.getBinary(FileAssetAPI.BINARY_FIELD).toPath());
+        
+        assertTrue("TitleImage read correctly", Arrays.equals(fileContents ,txtBlah));
+
+        
+        
+    }
+    @Test
+    public void test_titleImage_Field() throws Exception{
```

Comment:
@wezell  We generally try to follow any of these test-method naming convention: https://dzone.com/articles/7-popular-unit-test-naming

Reply:
Good to know, I changed them